### DISCUSSION TITLE
Discoverability/Tags

### DISCUSSION THREAD

**Discussion by josh-hemphill** [nestdotland/nest.land#316] [Ideas]
I couldn't find if/where this has been mentioned.
deno.land has a bit of a discoverability problem, and nest.land isn't much better. Searching only seems to match on title, and when trying to find something like an ORM or database module, or any other category, and find something that's being actively used, it's kind of a nightmare. 
Since nest.land has the advantage of supporting an `egg.json` file, tags could be implemented agnostic of any repository (If deno.land were to implement it, I'd think they'd have to just pull from github tags).

**Comment by SteelAlloy** [comment]
Hi @josh-hemphill 👋 
There hasn't been much activity on our side lately, but we continue to think about how to improve nest.land.
Keyword discovery is one of the planned features, you will be able to add those in the configuration files.

See https://github.com/nestdotland/nest/blob/08b8c1d72adf8ed4a76d865d1af63ebe104b21b5/src/mod/utils/types.ts#L29
However all of this is still a work in progress, we don't have a release date.
